src/projects/EnsureThat/EnsureComparableExtensions.cs

```diff
@@ -4,6 +4,8 @@
 
 namespace EnsureThat
 {
+    using System.Collections.Generic;
+
     public static class EnsureComparableExtensions
```

Comment:
This functionality does not need to be added. The `Ensure.That(x,y)` flavor is going away so don't really think features has to be added to that part.

Reply:
Just saw your other work with `Param<T>` being a struct. If that goes in it means that this should be kept.